frontend/src/static/js/main.js

```diff
@@ -99,10 +99,11 @@ window.app = new window.Vue({
 
     isLoading: false,
     isCollapsed: false,
+
+    // isTabActive used to force tab wrapper to load
     isTabActive: true,
```

Comment:
perhaps inline the comments? E.g. `isTabActive: true,  // to force tab wrapper to load`